build/media-suite_compile.sh

```diff
@@ -707,6 +708,16 @@ if [[ $standalone = y && $faac = y ]] && ! files_exist "${_check[@]}" &&
     do_checkIfExist
 fi
 
+_check=(bin-audio/exhale.exe)
+if [[ $standalone = y && $exhale = y ]] &&
```

Comment:
I'm not so sure on if the standalone check is necessary here since by answering y to exhale, they say yes to a `exhale binary`, but if they say no to standalone, then there's nothing that is produced by saying y to exhale